jaxrs/src/main/java/org/killbill/billing/jaxrs/json/OverdueConditionJson.java

```diff
@@ -57,7 +57,9 @@ public OverdueConditionJson(@JsonProperty("timeSinceEarliestUnpaidInvoiceEqualsO
     }
 
     public OverdueConditionJson(final OverdueCondition overdueCondition) {
-        this.timeSinceEarliestUnpaidInvoiceEqualsOrExceeds = new DurationJson(overdueCondition.getTimeSinceEarliestUnpaidInvoiceEqualsOrExceeds());
+        this.timeSinceEarliestUnpaidInvoiceEqualsOrExceeds = overdueCondition.getTimeSinceEarliestUnpaidInvoiceEqualsOrExceeds() == null ?
+                                                             new DurationJson(null, 0) :
+                                                             new DurationJson(overdueCondition.getTimeSinceEarliestUnpaidInvoiceEqualsOrExceeds());
         this.controlTagInclusion = overdueCondition.getInclusionControlTagType();
```

Comment:
IMO, `this.timeSinceEarliestUnpaidInvoiceEqualsOrExceeds` should be set to null if  `overdueCondition.getTimeSinceEarliestUnpaidInvoiceEqualsOrExceeds() == null`

Reply:
You're right, just set it to null works, too.